.github/workflows/operator-ci.yml

```diff
@@ -18,8 +23,8 @@ defaults:
     shell: bash
 
 concurrency:
-  # Only cancel jobs for PR updates
-  group: operator-ci-${{ github.ref }}
+  # Only cancel jobs for PR updates, prevent canceling run-matrix-with-quarkus-next by creating a separate group
+  group: operator-ci-${{ github.event.inputs.is-quarkus-next == 'true' && 'quarkus-next' || github.ref }}
```

Comment:
Could you please elaborate, why do we need this? Is this for the case when somebody pushes to the `quarkus-next` branch during nightly testing? It should be fine to cancel in that case – what's the point of waiting for obsolete test results? Maybe we don't even need to wait for the test results in the `quarkus-next` workflow, just "fire-and-forget". ;) It would've been the same if we used PAT.

Reply:
Correct, my intention was to separate nightly testing regardless of everything else. I thought that it could be useful mainly with regards to the team members shifts that we were discussing (in order to check the nightly run result every morning) or with regards to the Keycloak Dashboard results (which can be potentially influenced by the possible cancellation).

But I see your point, it's a corner case, which if we had ignored it, nothing serious would happen.

Reply:
Yeah, IMHO it's a corner case. We don't even handle for `main` branch (nightly testing that can be cancelled by a push), correct?

Reply:
let's ignore it then, suggestion applied.